static-instrumentation/README.md

```diff
@@ -0,0 +1,17 @@
+# OpenTelemetry Java static instrumentation
+
+## Structure
+
+### Agent instrumenter
+
+Module enhancing OpenTelemetry Java Agent for static instrumentation. The modified agent is capable of instrumenting and saving a new JAR with all relevant instrumentations applied and necessary helper class-code included.
+
+In order to statically instrument a JAR, modified agent needs to be both attached (`-javaagent:`) and run as the main method (`io.opentelemetry.javaagent.StaticInstrumenter` class).
```

Comment:
```suggestion
In order to statically instrument a JAR, modified agent needs to be both attached (`-javaagent:`) and run as the main method (`io.opentelemetry.javaagent.StaticInstrumenter` class).
```

Since we have control of main, can we find a way to run without duplicating into `javaagent:` arg?

Reply:
maybe using the [runtime-attach](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/runtime-attach) component?

Reply:
Yeah, good idea, let's do it once we have actual code (and change the instructions then).